XRTK-Core/Packages/com.xrtk.core/Definitions/Utilities/ProfileMenuItemIndices.cs

```diff
@@ -11,6 +11,7 @@ public enum CreateProfileMenuItemIndices
         Configuration = 0,
         Camera,
         Input,
+        InputSimulation,
```

Comment:
Can we add this to the end of the enum, so as not to break current serialization

Reply:
There isn't anything serialized with these. It's okay to rearrange as needed. It's only sorting the menu items as they appear in the editor menus, so there's no worry here.

Please leave as is if it's intended that Input Simulation be found under Input.